.circleci/config.yml

```diff
@@ -298,3 +333,12 @@ workflows:
                 core_dependencies: [false, true]
                 codecov: [true]
             name: "linux python << matrix.python_version >> unit tests, core dependencies << matrix.core_dependencies >>, codecov << matrix.codecov >>"
+  build_conda_pkg:
+    jobs:
+      - shellcheck/check
+      - build_conda_pkg:
+          requires:
+            - shellcheck/check
+          filters:
+            branches:
+              only: main
```

Comment:
This means that this will run after a PR gets merged to `main`.

We created this check to catch breaking changes to our conda recipe (like new dependencies or dependency updates) before release day and having it run after a merge accomplishes that without having to slow down development or consume too many ci resources.

Reply:
very cool!

Reply:
SGTM!